sw/device/lib/dif/dif_entropy_src_unittest.cc

```diff
@@ -597,7 +599,7 @@ TEST_F(ObserveFifoWriteTest, FifoFull) {
         kMultiBitBool4True},
        {ENTROPY_SRC_FW_OV_CONTROL_FW_OV_MODE_OFFSET, kMultiBitBool4True}});
   EXPECT_READ32(ENTROPY_SRC_FW_OV_WR_FIFO_FULL_REG_OFFSET, 1);
-  EXPECT_EQ(dif_entropy_src_observe_fifo_write(&entropy_src_, buf, 4),
+  EXPECT_EQ(dif_entropy_src_observe_fifo_write(&entropy_src_, buf, 4, nullptr),
```

Comment:
This is a good opportunity to test the use of this new argument, by passing a valid pointer and check if the value matches the expected.

Reply:
A good point.  I will add that!

Reply:
Done!